js/foam/browser/u2/BrowserView.js

```diff
@@ -19,431 +19,360 @@ CLASS({
   name: 'BrowserView',
   extends: 'foam.u2.View',
   requires: [
-    'foam.browser.BrowserConfigU2',
-    'foam.browser.u2.StackView',
-    'foam.input.touch.GestureManager',
-    'foam.input.touch.TouchManager',
-    'foam.u2.ElementParser',
+    'foam.u2.DAOController',
+    'foam.u2.DAOCreateController',
+    'foam.u2.PropertyView',
+    'foam.u2.SearchBorder',
     'foam.u2.md.ActionButton',
-    'foam.u2.md.Checkbox',
     'foam.u2.md.Input',
     'foam.u2.md.Select',
-    'foam.u2.md.SharedStyles',
-    'foam.u2.md.TextArea'
   ],
-
   imports: [
-    'gestureManager',
-    'touchManager',
+    'document',
+    'headerColor',
+    'menuFactory',
+    'selection$',
+    'stack',
   ],
   exports: [
-    'gestureManager',
-    'stack',
-    'touchManager',
+    'data as dao',
```

Comment:
Do we actually need this?

All the DAOs are exported by name, it might be better to let models import the specific dao they need always.